### PR TITLE
Add styles for #33741. Remove boards and add underlines.

### LINKED ISSUE #35201: [BITV] Focused item overlaps neighbor elements with text and border
> * Keyboard focus overlaps with text

@jancborchardt  you mean 

Screenshot from 2022-11-15 11-27-55 ?

Then i guess we need another design

### PR DESCRIPTION
**Pull request by JuliaKirschenheuter**
* Resolves: https://github.com/nextcloud/server/issues/35201

## TODO

- Add styles which were not in PR https://github.com/nextcloud/server/pull/35501 after rebasing.
- See: https://github.com/nextcloud/server/pull/35501#issuecomment-1333829401

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [x] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes => screenshots are here https://github.com/nextcloud/server/pull/35501
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by szaimen — APPROVED** [review]

**Comment by szaimen** [comment]
`node` is failing...

**Review by Pytal — APPROVED** [review]

**Comment by szaimen** [comment]
@JuliaKirschenheuter looking at what the compile bot did did you forget to apply your changes to server.scss?

**Comment by JuliaKirschenheuter** [comment]
> looking at what the compile bot did did you forget to apply your changes to server.scss?

I think i haven't forgotten, but in some point after local rebasing i've lost them. I will check it carefully in a feature.

**Comment by ChristophWurst** [comment]
@JuliaKirschenheuter see https://github.com/nextcloud/server/pull/35545/files

**Comment by ChristophWurst** [comment]
You have to make your changes on the source *.scss* files. The *.css* is the output.

See the https://github.com/nextcloud/server/blob/master/core/css/server.scss for the included styles. One of those includes will have the rules you adjusted.

**Review by szaimen — DISMISSED** [review]
Blocking:
> You have to make your changes on the source _.scss_ files. The _.css_ is the output.

**Comment by szaimen** [comment]
@JuliaKirschenheuter can you try to add your logic into the block above? It looks like this is now dublicating a lot of the logic...

File: core/css/header.scss [hunk 1]
```diff
@@ -21,10 +21,35 @@
...
+		width: 12px;
+		height: 2px;
+		border-radius: 3px;
+		background-color: var(--color-primary-text);
+		left: 50%;
+		opacity: 1;
+	}
+
+	a:not(.button):focus-visible::after, .button-vue:focus-visible::after {
+		bottom: 2%;
```
Comment (ChristophWurst) [inline core/css/header.scss:40]: why percent and not pixels, vh or vw?

**Review by ChristophWurst — APPROVED** [review]
Code looks :+1:

**Pull request merged**